toonz/sources/toonz/flipbook.cpp

```diff
@@ -272,9 +272,9 @@ void FlipBook::addFreezeButtonToTitleBar() {
         titleBar, ":Resources/pane_freeze_off.svg",
         ":Resources/pane_freeze_over.svg", ":Resources/pane_freeze_on.svg");
     m_freezeButton->setToolTip("Freeze");
-    titleBar->add(QPoint(-55, 0), m_freezeButton);
+    titleBar->add(QPoint(-64, 0), m_freezeButton);
     connect(m_freezeButton, SIGNAL(toggled(bool)), this, SLOT(freeze(bool)));
-    QPoint p(titleBar->width() - 62, 1);
+    QPoint p(titleBar->width() - 62, 0);
```

Comment:
Thank you for finding and tackling with the offset bug.
Can you modify this line as follows?
`QPoint p(titleBar->width() - 64, 0);`

For now it seems that the freeze button slightly offset (2pixel) to the left when resizing the window.
It is because of the mismatch of the horizontal position set at here and line 275. Thanks!

Reply:
Now it makes sense, thanks @shun-iwasawa! Changes made.